infra/k8s/redis-exporter-values.yaml

```diff
@@ -0,0 +1,67 @@
+rbac:
+  # Specifies whether RBAC resources should be created
+  create: true
+  # Specifies whether a PodSecurityPolicy should be created
+  pspEnabled: true
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+replicaCount: 1
+image:
+  repository: oliver006/redis_exporter
+  tag: v1.3.4
+  pullPolicy: IfNotPresent
+extraArgs: {}
+# Additional Environment variables
+env: {}
+# - name: REDIS_PASSWORD
+#   valueFrom:
+#     secretKeyRef:
+#       key: redis-password
+#       name: redis-config-0.0.2
+service:
+  type: ClusterIP
+  port: 9121
+  annotations: {}
+  labels: {}
+resources: {}
+redisAddress: redis://redis-master:6379
+annotations:
```

Comment:
Based on what I read, adding those `annotations` should tell `prometheus` to scrape that endpoint, but this doesn't seem to be working, so I've done something wrong.

Reply:
Note to self - read the full values.yaml, no matter how long they are (facepalm)